src/deferred.js

```diff
@@ -157,6 +157,10 @@ jQuery.extend( {
 											mightThrow();
 										} catch ( e ) {
 
+											if ( jQuery.Deferred.exceptionHook ) {
+												jQuery.Deferred.exceptionHook( e );
```

Comment:
Calling it `exceptionHook` doesn't change the fact that it is invoked for every Promise-like rejection.